photocolle-sdk/photocolle-sdk.framework/README.mkd

```diff
@@ -1,31 +0,0 @@
-## INSTALLATION
```

Comment:
i am removing the folder along with the contents.
the separated PR also consist of rearranging readmes placement.